relay-server/src/actors/upstream.rs

```diff
@@ -204,16 +272,14 @@ impl UpstreamRelay {
         }
     }
 
-    fn send_request<P, F>(
-        &self,
-        method: Method,
-        path: P,
-        build: F,
-    ) -> ResponseFuture<ClientResponse, UpstreamRequestError>
-    where
-        F: FnOnce(&mut ClientRequestBuilder) -> Result<ClientRequest, ActixError>,
-        P: AsRef<str>,
-    {
+    fn send_http_request(&mut self, request: UpstreamRequest) -> ResponseFuture<(), ()> {
```

Comment:
_nit_: Rename this to `send_request` and the other function to `enqueue_request` for clarity.

Reply:
ok